### PR TITLE
DATAUP-728: Removing unused files and directories

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Removing some old files that are no longer in use, and updating files that mention the unused files and directories to remove mentions of those files/dirs. The files in question are not called by the codebase and/or reference other files that are long deleted.

Directories removed:
- kbase-extension/static/kbase/js/patched-components
- nbextensions/appCell2/python

# Jira Ticket / Issue #

Related Jira ticket: https://kbase-jira.atlassian.net/browse/DATAUP-728
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [ ] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook
- [x] (Python) I have run Black and Flake8 on changed Python code manually or with a git precommit hook

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,8 +1,12 @@
 .git
 .github
+.husky
```
Comment (ialarmedalien) [inline .eslintignore:3]: edit the .eslintignore to cover the same files as the .prettierignore, and sort the lines alphabetically.

**Review by eapearson — CHANGES_REQUESTED** [review]
Looks fine to me from visual inspection. Funny patched select2 component!

Since GHA tests are not working, perhaps you could add a comment that tests run fine for you locally, even though the GHA tests are timing out?

**Review by eapearson — CHANGES_REQUESTED** [review]
Also, an addition to release notes would be nice.

**Comment by ialarmedalien** [comment]
Re: release notes: this is part of a series of PRs so will update release notes at the end.

**Review by briehl — APPROVED** [review]
Looks fine. Goodbye, unused patchwork!

**Pull request merged**